app/build.gradle.kts

```diff
@@ -273,6 +273,8 @@ dependencies {
     implementation("com.facebook.shimmer:shimmer:0.5.0")
 
     // test
+    testImplementation("junit:junit:4.13")
+    testImplementation("com.google.truth:truth:1.1.2")
```

Comment:
Should we check out using `assertk` over `truth`? The former seems to leverage Kotlin's features a bit more, wdyt?

Reply:
Looked into `assertk`, its nice that they leverage Kotlin more but they are at version `0.23.1` at the moment while `truth` is at `1.1.2`. `truth` is also the framework [suggested](https://developer.android.com/training/testing/unit-testing/local-unit-tests) and built by Google which I think is a plus. And they also seem to support the framework well. 

I propose we run with `truth` for now, but if we feel that its too clunky with Kotlin we can change to `assertk`, sounds reasonable?

Reply:
I think `assertk` is actually stable, despite the `0.*`-version 😉 Also, we already have `assertk` in the project, I think it could be valuable to not diverge on too many assertion-solutions 😄

Reply:
Ah wasnt aware we already used `assertk`. Ill remove `truth`!